src/settings.py

```diff
@@ -155,12 +131,10 @@ def migrate_2_to_3(self):
             del self.stratagemKeyDelayJitter
         print("Settings migrated to version 3. Remember to save.")
 
-class Loadout:
-    """
-    Simple data class designed to handle the storage and management of different 
-    configurations of macro keys for a particular setup.
-    This class is used primarily in the settings.
-    """
-    def __init__(self, name, macroKeys):
-        self.name = name
-        self.macroKeys = macroKeys
\ No newline at end of file
+    def migrate_3_to_4(self):
+        self.version = 4
+    
+        if hasattr(self, 'loadouts'):
+            delattr(self, 'loadouts')
+
+        print("Settings migrated to version 4. Remember to save.")
```

Comment:
Should we add migration to the loadouts file?

Reply:
Can use same as settings, however, until it is needed, it would just be messy.. so omitted.